### PR TITLE
AST-1007 - Fix: Image + Text blocks - Alignment not working properly with Image size - Medium for theme.json in the editor

### PR DESCRIPTION
**Pull request by premanshup**
### Description
Image + Text - Alignment not working properly with Image size - Medium for theme.json in the editor. This issue is caused due to the float properties changed in the WP 5.8 as mentioned [here](https://github.com/WordPress/gutenberg/issues/33385#issuecomment-893551418).

### Screenshots
Editor - [https://share.bsf.io/2Nulnygy](https://share.bsf.io/2Nulnygy) ( with theme.json )
Editor - [https://share.bsf.io/Wnu0or20](https://share.bsf.io/Wnu0or20) ( without theme.json -> correct )

### Steps to reproduce
1. Add image block
2. Set Image Size as Medium -> https://share.bsf.io/YEuOLyNP
3. Update alignment of the image to left.
4. Add paragraph text below
5. You can see the issue, the text is not right aligned, rather it is below image block which is not the expected behavior.

### Types of changes

Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Tested by adding image + text block in the editor with theme.json.

### Checklist:
- [ ] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Comment by imnavanath** [comment]
@premanshup can you please add changelog for this.

**Comment by premanshup** [comment]
> @premanshup can you please add changelog for this.

@imnavanath Added!

**Comment by imnavanath** [comment]
> @imnavanath Added!
Thanks, accordingly I've updated changelog P2. 👍

File: inc/core/class-gutenberg-editor-css.php [hunk 1]
```diff
@@ -364,6 +364,15 @@ public static function get_css() {
...
 				);
 				$desktop_css['.ast-page-builder-template .editor-styles-wrapper, .ast-plain-container .editor-styles-wrapper'] = $background_style_data;
+				$desktop_css['.wp-block[data-align=left]>*'] = array(
+					'float' => 'left',
+				);
+				$desktop_css['.wp-block[data-align=right]>*'] = array(
+					'float' => 'right',
+				);
+				$desktop_css['.wp-block[data-align=left], .wp-block[data-align=right]'] = array(
+					'float' => 'none !important',
```
Comment (imnavanath) [inline inc/core/class-gutenberg-editor-css.php:374]: Is there any feasibility @premanshup, if we can avoid `!important` ?

Comment (premanshup) [reply inc/core/class-gutenberg-editor-css.php:374]: @imnavanath Nope!

**Review by sagarb90 — APPROVED** [review]

**Pull request merged**